### DISCUSSION TITLE
Beta instead of Pro

### DISCUSSION THREAD

**Discussion by mjmaher987** [community/community#64931] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've updated my github from "Beta" to "Pro" using Student Developer Pack, but it is still showing "Beta" for me.
What's the problem?

**Comment by mjmaher987** [comment]
It may be the delay bug from the GITHUB.

**Comment by mjmaher987** [comment]
![image](https://github.com/community/community/assets/77095635/8c90db1a-48be-4cd4-b0e6-325aad519ba2)
This is the verification of github. I think it is worth attention to update the github status more rapidly.

**Comment by mjmaher987** [comment]
![image](https://github.com/community/community/assets/77095635/f3b595ea-8fe1-4f50-82f6-02a5a904f118)
As you can see in the picture, it is still "Beta", Not "Pro".

**Comment by Piero24** [comment]
@mjmaher987 That's not for the Github subscription. The Beta is related to the various Badges because they are relatively new so they are still in beta. The Student Developer Pack will generally be accepted within a few days and within a week of approval you will be awarded the "pro badge" on your profile. with the related benefits. Timings may vary from period to period as there may be periods with higher demands.

[See here](https://github.com/orgs/community/discussions/66773#discussion-5615588)
